package.template.json

```diff
@@ -55,7 +62,10 @@
     "metro-react-native-babel-preset": "0.71.3",
     "patch-package": "6.4.7",
     "prettier": "1.19.1",
-    "react-test-renderer": "17.0.2"
+    "react-test-renderer": "17.0.2",
+    "plist": "3.0.5",
+    "xcode": "3.0.1",
```

Comment:
```suggestion
```

`plist` and `xcode` are also provided in `scripts/package.json`.

Reply:
this did not work for me, i have the node_modules installed in scripts folder, but when running build scripts it tried to find the dependencies in root node_modules folder, so i added the required dependencies